zap/zap.yaml

```diff
@@ -54,46 +33,63 @@ jobs:
   # load our authentication and httpsender script
   - type: script
     parameters:
-      action: 'add'
-      type: 'httpsender'
-      engine: 'Graal.js'
-      name: 'add_bearer_token'
-      file: 'scripts/httpsender/add_bearer_token.js'
+      action: add
+      type: httpsender
+      engine: Graal.js
+      name: add_bearer_token
+      file: scripts/httpsender/add_bearer_token.js
   - type: script
     parameters:
-      action: 'add'
-      type: 'authentication'
-      engine: 'Graal.js'
-      name: 'get_bearer_token'
-      file: 'scripts/authentication/get_bearer_token.js'
+      action: add
+      type: authentication
+      engine: Graal.js
+      name: get_bearer_token
+      file: scripts/authentication/get_bearer_token.js
+
+  # configure the passive scan
+  - type: passiveScan-config
+    parameters:
+      maxAlertsPerRule: 10
+      scanOnlyInScope: true
+    rules:
```

Comment:
Could you add a quick one-line comment to each of these rules to summarise why they have been disabled so that the next person that sees it doesn't try to re-add them?

Reply:
Comments added.